docs/hacking/chrome-extension.rst

```diff
@@ -0,0 +1,60 @@
+=============================
+Building the Chrome extension
+=============================
+
+You can build a local copy of the `Hypothesis Chrome extension`_ based on the
+current contents of your working tree and install it in Chrome for development
+or testing. To build the Chrome extension:
+
+.. _Hypothesis Chrome extension: https://chrome.google.com/webstore/detail/hypothesis-web-pdf-annota/bjfhmglciegochdpefhhlphglcehbmek
+
+1. Do an :doc:`h development install </hacking/install>`.
+
+2. Build the Chrome extension locally with a fake ID.
+
+   The ``hypothesis-buildext`` command requires the ID that Chrome will give to
+   the extension for the ``--assets`` argument, but Chrome won't generate the
+   ID until after you’ve built the extension and loaded it into Chrome! It's
+   therefore necessary to build the extension once with a fake ID and then
+   rebuild it with the correct one.
+
+   Once it has generated an ID for your local build of the extension Chrome
+   will always use the same ID (even when you rebuild and reload the
+   extension), so you'll only have to do this once:
+
+   .. code-block:: bash
+
+      hypothesis-buildext conf/development.ini chrome
+          --base   'http://127.0.0.1:5000'
+          --assets 'chrome-extension://notarealid/public'
```

Comment:
Hmm. This isn't how I usually build a dev extension. Just omit `--assets` and it will load assets from the application. That gets around the need to build everything twice.